ui/app/src/views/ViewDashboard.tsx

```diff
@@ -161,6 +161,10 @@ function ViewDashboard() {
   }, [actionRef, navigate, projectName]);
 
   if (isLoading) return null;
+  if (error !== null) {
```

Comment:
``if (error && actionRef.current !== CreateAction) `` ? this is it @Gladorme ?

Reply:
Yes